tools/cibsecret.in

```diff
@@ -171,7 +171,7 @@ check_env() {
     else
         fatal $CRM_EX_NOT_INSTALLED "please install pssh, pdsh, or ssh to run $PROG"
     fi
-    ps -ef | grep '[p]acemaker-controld' >/dev/null ||
+    ps -efww | grep '[p]acemaker-controld' >/dev/null ||
```

Comment:
I didn't realize we were using `-ef` -- that won't work correctly on BSD, which we try to support when possible. Let's use `ps axww` here instead, which should work on both BSD and Linux.